src/components/Header/WindowsBar.scss

```diff
@@ -0,0 +1,45 @@
+.window-controls {
+	align-items: center;
+	display: flex;
+	-webkit-app-region: no-drag;
+	justify-content: space-evenly;
+}
+
+.window-controls .button {
+	position: relative;
+	background-color: transparent;
+	width: 50px;
+	height: 100%;
+	transition: 200ms;
+
+	& path {
+		transition: 200ms;
+		transition-timing-function: ease-in-out;
+	}
+}
+
+.close-button:hover {
+	background-color: red;
```

Comment:
True Windows red ?

Reply:
I think i'm going to use color picker for that